### PR TITLE
lps-116858-1

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LPS-116858

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test

**Comment by zhangyan0701** [comment]
ci:test

**Comment by zhangyan0701** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path [hunk 1]
```diff
@@ -60,7 +60,7 @@
 </tr>
 <tr>
 	<td>ANY</td>
-	<td>//button[contains(.,'${key_text}')]</td>
+	<td>//a[contains(.,'${key_text}')] | //button[contains(.,'${key_text}')]</td>
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:63]: did you try something like ?
//*[contains(.,'${key_text}') and contains(@class, 'btn')]

Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:63]: Yes, I tried, but not all buttons have class 'btn', that's why we updated this locator to https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path#L66-L70 before.

And if use //*[contains(.,'${key_text}')], there will be more than one elements match it in some pages, but I didn't find out a common class or something else in both kinds of buttons (//button or //a).

**Comment by zhangyan0701** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path (hunk 1 above)
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path]: My changes caused tests failed at sign in. I will try the path that you mentioned, and will create a new locator for the button that doesn't have 'btn' class. Thanks for the help!

**Comment by zhangyan0701** [comment]
ci:test

**Comment by zhangyan0701** [comment]
ci:test

**Pull request closed without merging**